include/dlaf/communication/kernels/p2p.h

```diff
@@ -0,0 +1,72 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+/// @file
+
+#include <mpi.h>
+
+#include "dlaf/common/callable_object.h"
+#include "dlaf/common/pipeline.h"
+#include "dlaf/communication/communicator.h"
+#include "dlaf/communication/message.h"
+#include "dlaf/matrix/tile.h"
+#include "dlaf/sender/traits.h"
+#include "dlaf/sender/transform_mpi.h"
+#include "dlaf/sender/when_all_lift.h"
+
+namespace dlaf {
+namespace comm {
```

Comment:
I'm changing these in the other `communication/kernels` files at the moment and can do this later as well, but it could be updated here as well:
```suggestion
namespace dlaf::comm {
```